src/compo/viirs_l1bnc2ioda.py

```diff
@@ -158,6 +166,12 @@ def _read(self):
                 orbit_ad[:] = 1
             self.varAttrs['satelliteAscendingFlag', metaDataName]['description'] = '0=descending, 1=ascending'
 
+            # NASA VIIRS stored reflectance need to divide by
+            # cosine of solar zenith angle to get true reflectance
+            sec_term = 1.
+            if self.divide_cos_solarza:
+                sec_term = 1. / np.cos(solar_za * np.pi / 180.)
```

Comment:
```suggestion
            if self.divide_cos_solarza sec_term = 1. / np.cos(solar_za * np.pi / 180.) else sec_term = 1.
```

Reply:
This change didn't work, so I updated the code to the traditional if-else structure.
Maybe that's the thing you suggested? @BenjaminRuston

Reply:
this is the new standard syntax,,, there are AI tools to debug it ;0)

let me run it again and get this correct